docs/backend_visualisation.md

```diff
@@ -0,0 +1,46 @@
+# Backend System Visualisation
+
+This diagram will give you a quick overview of what the backend system looks like.
+
+## What does the integration look like?
+![app_w_relation](https://user-images.githubusercontent.com/29667122/81039468-71bf2600-8eec-11ea-8a9f-149a9dda0a0f.png)
```

Comment:
These images can be stored on the repository itself, we can never be sure what are the longevity of these URLs that are taken from inline comments.

Reply:
Thanks, for your suggestion, @SanketDG. Storing the images in the repository is no question the most ideal way. The thing is for normal contributors like us who're not maintainers of the repository, we don't have access and can't upload files as we like. So, @isabelcosta and I viewed this as the next best thing as alternative. Hope this makes sense 😉

Reply:
@mtreacy002 Apologies, I do not seem to understand. You can always put the images in the working directory of the git repository and then just git push them so that they appear here, and then they can be referenced from the README files

Reply:
Sorry, @SanketDG if I confused you. @isabelcosta and I discussed this matter (the best way to put image on repository/.md files) [here](https://github.com/anitab-org/mentorship-backend/pull/546#issuecomment-622364197). Instead of uploading the files as binary files we decided the better way is to use reference link from images uploaded to comments like these ones here. Please let me know if this clear up your doubt. Or perhaps you have a better suggestion, we'd like to hear about it. 😉

Reply:
Right, gotcha! I can see how this is beneficial (lesser repository size, smaller clone times) but Git can handle binary files pretty well but it also has a tradeoff, (offline READMEs not a possibility) But yeah, if the image sizes are a problem, then it's better not to add them to the repository.